src/sinopiaServerSpoof.js

```diff
@@ -12,6 +12,14 @@ const transcribedTitleRt = require('../static/spoofedFilesFromServer/fromSinopia
 const varTitleRt = require('../static/spoofedFilesFromServer/fromSinopiaServer/resourceTemplates/VarTitle.json')
 const workTitleRt = require('../static/spoofedFilesFromServer/fromSinopiaServer/resourceTemplates/WorkTitle.json')
 const workVariantTitleRt = require('../static/spoofedFilesFromServer/fromSinopiaServer/resourceTemplates/WorkVariantTitle.json')
+const lccnRt = require('../static/spoofedFilesFromServer/fromSinopiaServer/resourceTemplates/LCCN.json')
```

Comment:
I understand we need these hardcoded sources for the Sinopia Server Spoof (and likely will need to continue to support those sources that are not available in QA)

Reply:
I think we'll be able to replace these spoofed hardcoded ones once we can talk to sinopiaServer ....